dotCMS/src/main/java/com/dotcms/graphql/exception/FieldGenerationException.java

```diff
@@ -0,0 +1,22 @@
+package com.dotcms.graphql.exception;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import graphql.schema.GraphQLFieldDefinition;
+
+/**
+ * Runtime Exception thrown when a {@link GraphQLFieldDefinition} can't be generated
+ */
+public class FieldGenerationException extends DotRuntimeException {
+
+    public FieldGenerationException(String message) {
+        super(message);
+    }
+
+    public FieldGenerationException(Throwable cause) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'cause' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)